### ISSUE TITLE
License Violations

### ISSUE THREAD

**Issue by colinoflynn** [ElectronicCats/faultycat#1]
Hello - this project is currently in violation of CC-SA-3.0 license that PicoEMP is released under and this project heavily remixed. Please rectify this ASAP.

The license requires you to:

Attribution — You must give [appropriate credit](https://creativecommons.org/licenses/by-sa/3.0/#), provide a link to the license, and [indicate if changes were made](https://creativecommons.org/licenses/by-sa/3.0/#). You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.

ShareAlike — If you remix, transform, or build upon the material, you must distribute your contributions under the [same license](https://creativecommons.org/licenses/by-sa/3.0/#) as the original.

The biggest issue here is reusing the majority of the text in the description without correctly attributing back to PicoEMP, along with similar issues on design (at least design is more of a remix than direct copy). But the remix of PicoEMP into FaultyCat should be a clear step, but it's very muddled right now. The remixed material should be shared under the same license per CC-SA-3.0.

This is required as it's possible/likely that FaultyCat is itself remixed in the future, and if that happens someone may not realize they were actually borrowing from PicoEMP. The link between all the projects should be clear for future users.

Thanks!

**Comment by sabas1080** [comment]
Hello @colinoflynn  

First of all, apologies for not executing the license correctly. We want to clarify that our intention was not to take the project's appropriation, we have made the corresponding changes, and we await your feedback.
Thank you so much for doing this great project. We admire your work and hope you like the changes we have made and the next ones we make to your project.

Best regards,
